client/src/main/java/org/apache/cloudstack/ServerDaemon.java

```diff
@@ -101,6 +103,8 @@ public class ServerDaemon implements Daemon {
     private String keystoreFile;
     private String keystorePassword;
     private String webAppLocation;
+    private int minThreads = 10;
+    private int maxThreads = 500;
```

Comment:
This is a small a nitpick, but you don't need to initialize these values, as in the `properties.getProperty` call you already inform these as default.

Reply:
will you change this @ahmedali6 ? No big deal if you don't, but if you do, you can also comment out the new properties for the same reason.